src/WingetCreateCLI/UserSettings.cs

```diff
@@ -53,6 +53,40 @@ public static bool TelemetryDisabled
             }
         }
 
+        /// <summary>
+        /// Gets or sets the owner of the winget-pkgs repository.
+        /// </summary>
+        public static string WingetPkgsRepoOwner
+        {
+            get
```

Comment:
Can simplify the gets to:

get => Settings.WingetPkgsRepo.Owner;

Reply:
changed to suggestion